store/tx_manager.go

```diff
@@ -66,6 +78,25 @@ func (txm *TxManager) CreateTx(to common.Address, data []byte) (*models.Tx, erro
 		return nil
 	})
 
+	if err != nil && strings.Contains(err.Error(), "nonce is too low") {
```

Comment:
Looks like this comes from Parity, not Geth. I see what you mean about strings being brittle though. If geth or parity  change their error message, or we need to support some third ethereum client with a new error message, this check won't work. Maybe just try something broader like a regex for ` /nonce .*too low/`?

Reply:
Updated with support for geth + parity